package-lock.json

```diff
@@ -1,8 +1,5110 @@
 {
```

Comment:
seems like we can just remove this file. we have both a yarn.lock and a package-lock.json file